README-PDF.md

```diff
@@ -0,0 +1,40 @@
+# Advanced-Reports: PDF Reports
+
+PDF reports generate a PDF file from a *Source Report* (which provides the data to include in the
+report) and the *HTML Source* (which provides the document template/layout).
+
+## Source Report
+
+The source report for a PDF report can be any Instrument Query.
+
+## Paper Size
+
+Choose the paper size and orientation for your PDF file.
+
+## HTML Source
+
+Create your document template using HTML. The module will convert this to a PDF.
+
+You can use *field variables* and *logic tags* to include the data from the source report into your
+PDF report.
+
+**Field Variables** consist of the name of the field or column as output by the source report,
+enclosed in square brackets, for example `[record_id]`. Note that if the source report has square
+brackets in the column name, these will also have to be included, for example:
+`[[baseline][record_id]]`.
+
+**Logic Tags** control how the report is displayed:
+* `<?loop>` will repeat the following section, once for each row in the source report
+* `<?if(logic)>` where `logic` is REDCap calculation logic, will only show the following section if
+  the logic evaluates to true
+  * The logic can use any smart variable which does not require a record context, and query string
+    smart variables are also supported - for more information about query string smart varibles
+    please refer to the  [Instrument Query report instructions](README-InstrumentQ.md).
+* `<?end>` ends the previous *loop* or *if* section
```

Comment:
I would suggest adding a very simple example including the loop and if statement.,